ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockImporter.java

```diff
@@ -81,4 +81,20 @@ BlockImportResult importBlockForSyncing(
       HeaderValidationMode headerValidationMode,
       HeaderValidationMode ommerValidationMode,
       BodyValidationMode bodyValidationMode);
+
+  /**
+   * Attempts to import the given syncBlock. Uses "fast" validation. Performs light validation using
```

Comment:
The note about validation isn't right; this doesn't do any validation

Reply:
Done